test/appium/tests/atomic/chats/test_public.py

```diff
@@ -435,6 +435,35 @@ def test_different_status_in_timeline(self):
 
         self.errors.verify_no_errors()
 
+    @marks.testrail_id(700727)
+    @marks.medium
+    def test_gap_in_public_chat(self):
+        self.create_drivers(2)
+        device_1, device_2 = SignInView(self.drivers[0]), SignInView(self.drivers[1])
+        home_1, home_2 = device_1.create_user(), device_2.create_user()
+        message = "testing gap"
+        chat_name = home_1.get_random_chat_name()
+        profile_1 = home_1.profile_button.click()
+        for element in [profile_1.sync_settings_button, profile_1.sync_history_for_button, profile_1.element_by_translation_id("two-minutes")]:
```

Comment:
better don't have a loop here for best visibility

Reply:
thanx @churik , fixed